src/autoscaler/eventgenerator/generator/evaluator_test.go

```diff
@@ -579,11 +579,12 @@ var _ = Describe("Evaluator", func() {
 					database.RetrieveAppMetricsStub = func(appId string, metricType string, start int64, end int64) ([]*models.AppMetric, error) {
 						return appMetricGTBreach, nil
 					}
-					Expect(triggerChan).To(BeSent(triggerArrayGT))
 				})
 				Context("when the send request encounters error", func() {
 					JustBeforeEach(func() {
 						scalingEngine.Close()
+						Eventually(scalingEngine.HTTPTestServer).Should(BeNil())
```

Comment:
As we are not sending the request directly to Scaling Engine (to check the failure) rather to Evaluator, does this mean that  we should Eventually send the trigger to evaluator channel without waiting for the server to be nil?